website/src/content/docs/index.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: Welcome to Starlight
```

Comment:
I'm a bit puzzled. Some pages refer to SCAAML, others seem to be default/example ones from the Astro sometimes embedding resources such as a webp file that we don't need.